java/source/org/libreoffice/ide/eclipse/java/JavaWizardPage.java

```diff
@@ -109,23 +107,9 @@ public UnoFactoryData fillData(UnoFactoryData pData) {
     public void createControl(Composite pParent) {
 
         Composite body = new Composite(pParent, SWT.NONE);
-        body.setLayout(new GridLayout(2, false));
+        body.setLayout(new GridLayout(1, false));
         body.setLayoutData(new GridData(GridData.FILL_BOTH));
 
-        // Create the Java version row
-        mJavaVersionRow = new ChoiceRow(body, JAVA_VERSION,
-            Messages.getString("JavaWizardPage.JavaVersion"), null, false); //$NON-NLS-1$
```

Comment:
Please also remove the strings (search for the key "JavaWizardPage.JavaVersion" etc).